src/libstore/daemon.cc

```diff
@@ -308,8 +308,7 @@ static void performOp(TunnelLogger * logger, ref<Store> store,
 
     case wopQueryReferences:
     case wopQueryReferrers:
-    case wopQueryValidDerivers:
-    case wopQueryDerivationOutputs: {
```

Comment:
Do you know what the policy is on supporting on protocol versions? I don't!

Reply:
The daemon should support older clients. So we can't remove APIs like this.

Reply:
Woops sorry, I'll restore them